src/index.html

```diff
@@ -11,7 +11,182 @@
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+            autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="11"
+            maxlength="30"
+            autocomplete="off"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="yes"
+            >
+            Yes
+          </label>
+          
+          <label>
+            <input
+              type="radio"
+              name="cats"
+              value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label for="favcolor">
+            What is your favorite color?
+          </label>
+          <input
+            type="color"
+            id="favcolor"
+            name="favcolor"
+          >
+        </div>
+
+        <form class="form-field">
+          <label for="appt">
+          What time do you go to bed?
+          </label>
+          <input
+            type="time"
+            id="appt"
+            name="appt"
+            required
+          >
+        </form>
+
+        <p>
+          <label class="form-field">
+            What are your favorite brand of cars?
+            <select name="cars" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </label>
+        </p>
+
+        <div class="form-field">
+          <label for="rate">How do you rate or work?</label>
+          <input
+            type="range"
+            id="rate"
+            name="rate"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-group">
+        <legend>Additional info:</legend>
+
+        <div class="form-field">
+          <label for="comments">Comments:</label>
+          <textarea name="comments" 
+            id="comments" 
+            cols="20" 
+            rows="2"
+          >
+          </textarea>
+        </div>
+
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recommend"
+            required
+            autocomplete="off"
+          >
```

Comment:
recheck indentations and formating everywhere, according to checklist
```suggestion
          <select 
            name="recommend"
            required
            autocomplete="off"
          >
```

Reply:
![image](https://user-images.githubusercontent.com/99331793/187875039-490bbffa-1a70-4dd2-9001-23d82e99e5da.png)